tests/php/Model/SiteTreeTest.php

```diff
@@ -470,46 +479,47 @@ public function testGetByLink()
         );
     }
 
-    public function testGetByLinkAbsolute()
```

Comment:
Why is this being removed?

Reply:
Looks like a mistake when I was resolving conflicts. Will add back in.

Reply:
Done